Modules/Bridge/NumPy/wrapping/PyBuffer.i.in

```diff
@@ -70,7 +70,7 @@
             if source:
                 source.UpdateLargestPossibleRegion()
 
-        arrayView = itkPyBuffer@PyBufferTypes@.GetArrayViewFromImage(image, keep_axes)
+        arrayView = itkPyBuffer@PyBufferTypes@.GetArrayViewFromImage(image, keep_axes, update)
```

Comment:
if you just updated the source (line 71), shouldn't there be no need to do it again in GetArrayViewFromImage ?